### PR TITLE
Migrate DownloaGitHubNugetPackagev1 to node10

### PR DESCRIPTION
**Pull request by denis-tikhomirov**
**Task name**: DownloadGithubNugetPackageV1

**Description**: this pull request provides updating the NpmAuthenticateV0 task and its migration to Node 10

**Documentation changes required:** (Y/N) N

**Added unit tests:** (Y/N) N

**Attached related issue:** (Y/N) N

**Checklist**:
- [x] Task version was bumped - please check [instruction](https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md) how to do it
- [ ] Checked that applied changes work as expected

### CODE REVIEW

**Comment by denis-tikhomirov** [comment]
Changes in the PR are prepared to migrate task DownloadGithubNugetPackageV1 to Node10.
@richakumar, @harshiniks, could you please help with testing of the changes? I see in code owners list that you are owners of DownloadGithubNugetPackageV1 task.

**Comment by denis-tikhomirov** [comment]
@mmrazik, please take a look at the PR.

**Review by EzzhevNikita — APPROVED** [review]

File: Tasks/DownloadGitHubNugetPackageV1/task.json [hunk 1]
```diff
@@ -17,10 +17,10 @@
     "demands": [],
     "version": {
         "Major": 1,
-        "Minor": 200,
-        "Patch": 1
+        "Minor": 202,
```
Comment (mmrazik) [inline Tasks/DownloadGitHubNugetPackageV1/task.json:20]: Can we bump the version here, please?

**Review by mmrazik — CHANGES_REQUESTED** [review]

File: Tasks/DownloadGitHubNugetPackageV1/task.json (hunk 1 above)
Comment (mmrazik) [reply Tasks/DownloadGitHubNugetPackageV1/task.json:20]: Please ping me via Teams once updated.

Comment (max-zaytsev) [reply Tasks/DownloadGitHubNugetPackageV1/task.json]: Updated to 206

**Review by mmrazik — APPROVED** [review]

**Review by v-nmursapall — APPROVED** [review]
Looks good. Approved

**Pull request merged**